### PR TITLE
Update content view CLI test

### LINKED ISSUE #2100: Automation failure : test_cv_associate_composite_dupe_modules_negative
TestName:
tests.foreman.cli.test_contentviews.TestContentView.test_cv_associate_composite_dupe_modules_negative

the test is failing due to a bz:
- https://bugzilla.redhat.com/show_bug.cgi?id=1214212

please use the decorator  @skip_if_bug_open('bugzilla', 1214212)
the test would be waived for the upcoming runs untill the bz is fixed

please fix it
thanks

### PR DESCRIPTION
**Pull request by elyezer**
Update test_cv_associate_composite_dupe_modules_negative in order to
check for all puppet modules returned and also for the number of
available puppet modules. This will ensure in the future that BZ
1222118 will happen again. Also skip the test due to the mentioned BZ.

Closes #2100

```
py.test tests/foreman/cli/test_contentviews.py -ktest_cv_associate_composite_dupe_modules_negative
=================================== test session starts ====================================
platform darwin -- Python 2.7.6 -- py-1.4.26 -- pytest-2.6.4
collected 57 items

tests/foreman/cli/test_contentviews.py s

======= 56 tests deselected by '-ktest_cv_associate_composite_dupe_modules_negative' =======
======================== 1 skipped, 56 deselected in 32.46 seconds =========================
```

### CODE REVIEW

File: tests/foreman/cli/test_contentviews.py [hunk 1]
```diff
@@ -965,35 +968,29 @@ def test_cv_associate_composite_dupe_modules_negative(self):
...
-        )
-        self.assertGreater(len(result.stderr), 0)
+            # Re-associate same puppet module to CV
+            result = ContentView.puppet_module_add({
+                u'content-view-id': content_view['id'],
+                u'name': puppet_module['name'],
+            })
+            self.assertNotEqual(
+                result.return_code, 0,
+                'An already added puppet module shoul not be added twice'
```
Comment (oshtaier) [inline tests/foreman/cli/test_contentviews.py]: 'should not be added twice'

**Comment by oshtaier** [comment]
ACK

File: tests/foreman/cli/test_contentviews.py (hunk 1 above)
Comment (elyezer) [inline tests/foreman/cli/test_contentviews.py]: Thanks

**Comment by Ichimonji10** [comment]
ACK

**Pull request merged**